src/main/flight/failsafe.c

```diff
@@ -208,14 +216,8 @@ void failsafeUpdateState(void)
                         failsafeState.receivingRxDataPeriodPreset = PERIOD_OF_1_SECONDS;    // require 1 seconds of valid rxData
                         reprocessState = true;
                     } else if (!receivingRxData) {
-                        if (millis() > failsafeState.throttleLowPeriod) {
-                            // JustDisarm: throttle was LOW for at least 'failsafe_throttle_low_delay' seconds
-                            failsafeActivate();
-                            failsafeState.phase = FAILSAFE_LANDED;      // skip auto-landing procedure
-                            failsafeState.receivingRxDataPeriodPreset = PERIOD_OF_3_SECONDS; // require 3 seconds of valid rxData
```

Comment:
Why was this removed?

This is to stop failsafe from kicking in if throttle was low for long enough, based on the assumption that in this case the craft has landed.

Reply:
For some reason, we were never able to get it to set the phase to RX_LOSS_DETECTED without the removal of this.  It was always returning true.  Will investigate further 👍 The state in failsafe.c is very hard to follow.